deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -0,0 +1,4770 @@
+const ARBITRUMONE_PSR = "0xF9263eaF7eB50815194f26aCcAB6765820B13D41";
+const ETHEREUM_CONVERTER_NETWORK = "0x232CC47AECCC55C2CAcE4372f5B268b27ef7cac8";
+const ETHEREUM_PRIME = "0x14C4525f47A7f7C984474979c57a2Dccb8EACB39";
+const ETHEREUM_PLP = "0x8ba6aFfd0e7Bcd0028D1639225C84DdCf53D8872";
+const ETHEREUM_PSR = "0x8c8c8530464f7D95552A11eC31Adbd4dC4AC4d3E";
+const OPBNBMAINNET_PSR = "0xA2EDD515B75aBD009161B15909C19959484B0C1e";
+const ARBITRUMSEPOLIA_PRIME = "0xadb04ac4942683bc41e27d18234c8dc884786e89";
+const ARBITRUMSEPOLIA_PLP = "0xe82c2c10f55d3268126c29ec813dc6f086904694";
+const ARBITRUMSEPOLIA_PSR = "0x09267d30798B59c581ce54E861A084C6FC298666";
+const SEPOLIA_PRIME = "0x2Ec432F123FEbb114e6fbf9f4F14baF0B1F14AbC";
+const SEPOLIA_PLP = "0x15242a55Ad1842A1aEa09c59cf8366bD2f3CE9B4";
+const SEPOLIA_PSR = "0xbea70755cc3555708ca11219adB0db4C80F6721B";
+const OPBNBTESTNET_PSR = "0xc355dEb1A9289f8C58CFAa076EEdBf51F3A8Da7F";
+const SEPOLIA_CONVERTER_NETWORK = "0xB5A4208bFC4cC2C4670744849B8fC35B21A690Fa";
+
+const ARBITRUMONE_GUARDIAN = "0x14e0E151b33f9802b3e75b621c1457afc44DcAA0";
+const ETHEREUM_GUARDIAN = "0x285960C5B22fD66A736C7136967A3eB15e93CC67";
+const OPBNBMAINNET_GUARDIAN = "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207";
+const SEPOLIA_GUARDIAN = "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb";
+const OPBNBTESTNET_GUARDIAN = "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf";
+const ARBITRUMSEPOLIA_GUARDIAN = "0x1426A5Ae009c4443188DA8793751024E358A61C2";
+
+const ETHEREUM_CONVERTERS: string[] = [
+  "0xaE39C38AF957338b3cEE2b3E5d825ea88df02EfE",
+  "0x4f55cb0a24D5542a3478B0E284259A6B850B06BD",
+  "0xcEB9503f10B781E30213c0b320bCf3b3cE54216E",
+  "0xDcCDE673Cd8988745dA384A7083B0bd22085dEA0",
+  "0xb8fD67f215117FADeF06447Af31590309750529D",
+  "0x1FD30e761C3296fE36D9067b1e398FD97B4C0407",
+];
+const SEPOLIA_CONVERTERS: string[] = [
+  "0xCCB08e5107b406E67Ad8356023dd489CEbc79B40",
+  "0x3716C24EA86A67cAf890d7C9e4C4505cDDC2F8A2",
+  "0x511a559a699cBd665546a1F75908f7E9454Bfc67",
+  "0x8a3937F27921e859db3FDA05729CbCea8cfd82AE",
+  "0x274a834eFFA8D5479502dD6e78925Bc04ae82B46",
+  "0xc203bfA9dCB0B5fEC510Db644A494Ff7f4968ed2",
+];
+
+enum AccountType {
+  NORMAL_TIMELOCK = "NormalTimelock",
+  FAST_TRACK_TIMELOCK = "FastTrackTimelock",
+  CRITICAL_TIMELOCK = "CriticalTimelock",
+}
+
+interface Permission {
+  params: string[];
+}
+
+interface Permissions {
+  [key: string]: Permission[];
+}
+
+const grantPermissions: Permissions = {
+  arbitrumone: [
+    {
+      params: [
+        ethers.constants.AddressZero,
+        "updateJumpRateModel(uint256,uint256,uint256,uint256)",
+        AccountType.NORMAL_TIMELOCK,
+      ],
+    },
+    {
+      params: [
+        ethers.constants.AddressZero,
+        "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+        AccountType.NORMAL_TIMELOCK,
+      ],
+    },
+    {
+      params: [
+        ethers.constants.AddressZero,
+        "setLastRewardingBlocks(address[],uint32[],uint32[])",
```

Comment:
Arbitrum one is a time base network, so I don't think we need this permission here